kalite/distributed/static/perseus/.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "ke"]
```

Comment:
Yeah, and .gitmodules only get read from the root directory of a git repo. So, using the subtree approach, this doesn't get looked at by our code.